app/controllers/chargeback_controller.rb

```diff
@@ -315,12 +475,16 @@ def cb_rates_delete
           page.replace("flash_msg_div", :partial => "layouts/flash_msg")
         end
       else
-        rates.push(params[:id])
+        rates_compute.push(params[:id])
       end
       cb_rate = ChargebackRate.find_by_id(params[:id])
-      process_cb_rates(rates, "destroy")  unless rates.empty?
+      process_cb_rates(rates_compute, "destroy") unless rates_compute.empty?
+      rates_storage = rates_compute.map { |v|; v.to_i + 1 } unless rates_compute.empty?
```

Comment:
Just curious, is there any reason we use `.to_i` here but not several lines above?

```
rates_storage = rates_compute.map { |v|; v + 1 }
...
rates_storage = rates_compute.map { |v|; v.to_i + 1 } unless rates_compute.empty?
```